documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -264,7 +264,7 @@ Alternatively you may define some parameters in constructor, and then set other
    proj.offset = [1,0,0];
 
 
-The most important fields of ``line_proj`` constructor are the fields, which define the 
+The most important fields of ``line_proj`` constructor are the fields, which define the
```

Comment:
```suggestion
The most important fields of ``line_proj`` constructor are the fields which define the
```
with this comma, there is no intent to limit the scope of fields to the defining scope. The which clause is merely an explanation of the term fields. The comma must be removed to limit the scope